lib/global_id/identification.rb

```diff
@@ -21,5 +21,31 @@ def to_signed_global_id(options = {})
     def to_sgid_param(options = {})
       to_signed_global_id(options).to_param
     end
+
+    ActiveRecord::Associations::Builder::BelongsTo.instance_eval do
```

Comment:
This should be moved to it's own module in a new file, like `association_identification.rb`, and then included in in the Railtie like `GlobalID::Identification` is.